images/ose-containernetworking-plugins.yml

```diff
@@ -15,6 +15,7 @@ from:
   builder:
   - stream: rhel-8-golang
   - stream: golang
+  - stream: rhel-8-golang
```

Comment:
Based on the error:
```
Build metadata for openshift/ose-container-networking-plugins expected 3 image parent(s), but the upstream Dockerfile contains 4 FROM statements. These counts must match. Detail: '[{'stream': 'rhel-8-golang'}, {'stream': 'golang'}, {'builder': [...], 'member': 'openshift-enterprise-base'}]' vs '['openshift/origin-release:rhel-8-golang-1.12', 'openshift/origin-release:rhel-7-golang-1.12', 'openshift/origin-release:rhel-8-golang-1.12', 'openshift/origin-base']'.
```
It seems to imply that we need 1 more `stream` to match with the `Dockerfile`. Though I would think this would have been a problem before too. Do we need one for `rhel-7-golang`?